tools/ui-components/tailwind.config.js

```diff
@@ -37,6 +39,17 @@ module.exports = {
       'default-background-danger-hover':
         'var(--default-background-danger-hover)',
       'default-background-info-hover': 'var(--default-background-info-hover)',
+      gray: {
+        0: 'var(--gray00)',
+        50: 'var(--gray05)',
+        100: 'var(--gray10)',
+        150: 'var(--gray15)',
+        450: 'var(--gray45)',
+        750: 'var(--gray75)',
+        800: 'var(--gray80)',
+        850: 'var(--gray85)',
+        900: 'var(--gray90)'
+      },
```

Comment:
I only need `gray90`, but adding all of the variables in anyway, since we already did that for other colors.